cmd/ore/aws/upload.go

```diff
@@ -236,6 +239,24 @@ func runUpload(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	tagMap := make(map[string]string)
+	if len(uploadTags) > 0 {
```

Comment:
Don't need the if, looping over an empty/nil list is a no-op

Reply:
:+1: Fixed it in latest push